specs/execution-api.md

```diff
@@ -3,10 +3,10 @@
 ## Overview
 
 The Execution API is the interface `Conductor` uses to drive deterministic
-execution of sequenced blocks in dependent chains. Inspired by other APIs, such
+derivation of sequenced blocks in rollup chains. Inspired by other APIs, such
```

Comment:
are we changing derivation to execution?

Reply:
I think for clarity purposes its best to use different verbiage than reflecting execution here, but then this line doesn't make sense... will update.